### PR TITLE
Added arm64 support

### LINKED ISSUE #29: Support for ARM64
I'm trying out the Docker image on my Macbook (with ARM CPU), and I'm seeing the following error:

> grafana-lgtm The requested image's platform (linux/amd64) does not match the detected host platform (linux/arm64/v8) and no specific platform was requested

I'm not seeing any traces are coming in, in Tempo, but I'm not sure if it's related (but I do not think so, since I've never had it working before :p).

Not sure it's already on the roadmap.

As a workaround I've added:

```yaml
  platform: linux/amd64
```

to my `compose.yaml` file.

### PR DESCRIPTION
**Pull request by devops-42**
This PR takes the architecture during the container build into account. It fixes some configuration issues at least I had when running the services. Should solve #29.

Any feedback welcome!

Cheers.

Fixes #29

### CODE REVIEW

File: docker/Dockerfile [hunk 1]
```diff
@@ -6,34 +6,37 @@ ENV TEMPO_VERSION="2.4.1"
 ENV LOKI_VERSION="2.9.6"
 ENV OPENTELEMETRY_COLLECTOR_VERSION="0.97.0"
 
+ARG TARGETARCH
```
Comment (zeitlinger) [inline docker/Dockerfile:12]: can you make `amd64` the default, similar to `RELEASE`?

**Review by zeitlinger — CHANGES_REQUESTED** [review]
thanks for the contribution :tada: 

@fstab please also check

File: docker/Dockerfile (hunk 1 above)
Comment (devops-42) [reply docker/Dockerfile:12]: Hi @zeitlinger,

thanks for your comment. IMHO, It shouldn't be necessary: The `TARGETARCH` is used as a kind of magic variable in Docker when building images. If the build is started with a Docker running on an `amd64` architecture, then `TARGETARCH` would set to `amd64` automatically.

But if there's a reason (pipeline, etc) I could do this of course :)

Cheers

Comment (zeitlinger) [reply docker/Dockerfile:12]: ah, didn't know!

Can you just add a comment explaining that's it's filled automatically?

Comment (devops-42) [reply docker/Dockerfile:12]: Have added some remarks regarding the usage of that variable.

**Comment by zeitlinger** [comment]
Thanks for the contribution!

**Pull request merged**